testdata/project-v2-multigroup/config/certmanager/certificate.yaml

```diff
@@ -1,16 +1,16 @@
 # The following manifests contain a self-signed issuer CR and a certificate CR.
 # More document can be found at https://docs.cert-manager.io
-# WARNING: Targets CertManager 0.11 check https://docs.cert-manager.io/en/latest/tasks/upgrading/index.html for 
+# WARNING: Targets CertManager 1.0.1 check https://docs.cert-manager.io/en/latest/tasks/upgrading/index.html for 
 # breaking changes
-apiVersion: cert-manager.io/v1alpha2
+apiVersion: cert-manager.io/v1
```

Comment:
v2 projects still been updated. It means that we are changing its template when we should not.

Reply:
Solved. Feel free to close this one.